auslib/test/test_db.py

```diff
@@ -898,36 +1270,36 @@ def testUpdateRule(self):
         what = dict(rules[0].items())
 
         what['mapping'] = 'd'
-        self.paths.updateRule(changed_by='bill', id_or_alias=1, what=what, old_data_version=1)
+        self.paths.update(where={"rule_id": 1}, what=what, changed_by="bill", old_data_version=1)
 
         rules = self.paths.t.select().where(self.paths.rule_id == 1).execute().fetchall()
         copy_rule = dict(rules[0].items())
         rule = self._stripNullColumns([copy_rule])
 
-        expected = [dict(rule_id=1, priority=100, backgroundRate=100, version='3.5', buildTarget='d', mapping='d', update_type='z', data_version=1)]
+        expected = [dict(rule_id=1, priority=100, backgroundRate=100, version='3.5', buildTarget='d', mapping='d', update_type='z', data_version=2)]
```

Comment:
I spent some time trying to find out, and I'm not sure :(. I may come back to this though...it's very concerned.